src/traceViewer/videoTileGenerator.ts

```diff
@@ -30,13 +30,24 @@ export class VideoTileGenerator {
   }
 
   async render(events: PageVideoTraceEvent[]) {
+    let ffmpegName = '';
+    if (process.platform === 'win32') {
+      if (process.arch === 'ia32')
+        ffmpegName = 'ffmpeg-win32';
```

Comment:
I remember a PSA that we drop win i32?

Reply:
i don't think we made that decision

Reply:
ah ye was FF only: https://playwright.slack.com/archives/CSUHZPVLM/p1596215581072600